src/demos/progress-indicator/progress-indicator-demo.component.html

```diff
@@ -94,15 +94,15 @@ <h2>Progress Indicator</h2>
   </sky-progress-indicator-reset-button>
 </sky-progress-indicator>
 <hr>
-<h2>Progress indicator passive</h2>
+<h2>Passive progress indicator</h2>
 <button
   class="sky-btn sky-btn-link"
   skyPopoverPlacement="right"
   skyPopoverTrigger="mouseenter"
   type="button"
   [skyPopover]="myPopover1"
 >
-  In progress
```

Comment:
I see where you're going, but the original text is most reflective of a real example. "In progress" would be the status of some item (expense report, grant proposal, etc.) and the passive progress indicator provides more detail. If we really want to improve this demo, I think there's two directions we could take. We could either _add_ context, by building a short "list" (2 items) of grant proposals, and putting passive progress indicators behind the status of one. Or, we could take the approach more consistent with how tightly-scoped most of our demos are and _remove_ that context (save it for the usage guidelines) by putting a passive progress indicator directly on the page, instead of inside a Popover.